File: 1480.java
/*Running Sum of 1d Array*/
class Solution {
    public int[] runningSum(int[] nums) {
        int[] x = new int[nums.length];
        x[0] = nums[0];
        for(int i = 1; i < nums.length; i++) {
            int f = nums[i] + nums[i-1]; 
            x[i] = f;
            nums[i] = x[i];
        }
        return x;
    }
}

File: 1480.py
# Running Sum of 1d Array
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        if len(nums) == 1:
            return [nums[0]]
        result = []
        for x in range(1,len(nums) + 1):
            result.append(sum(nums[:x]))
        return result
